test/snapshot_test.rb

```diff
@@ -0,0 +1,143 @@
+require_relative 'spec_helper'
+require 'yast'
+require 'dbus'
+require_relative '../src/lib/snapper/snapshot.rb'
+
+describe Yast2::Snapper::Snapshot, "with default strategy" do
```

Comment:
If I get strategy pattern, then this class you be independent on its implementation. So also its test should be indenpendent and if you want to test SnapShotDbusStrategy, then create separate test case